README.md

```diff
@@ -0,0 +1,64 @@
+# Avidemux
+
+Avidemux is a simple cross-platform video editor for Linux, Windows and MacOsX.
+
+# Download executables
+
+[Latest release](https://github.com/mean00/avidemux2/releases/latest)
+
+[Nightly builds](https://www.avidemux.org/nightly/)
+
+# Build from source
+
+Get the main repository:
+```
+git clone https://github.com/mean00/avidemux2.git
+cd avidemux2
+git submodule update --init --recursive
+```
+
+
+## Build on Linux
+
+Install build dependecies:
+> Debian / Ubuntu and variants:
+```
+bash createDebFromSourceUbuntu.bash --deps-only
+```
+> Fedora:
+```
+bash createRpmFromSourceFedora.bash --deps-only
+```
+Build Avidemux:
+```
+bash bootStrap.bash --with-system-libass
```

Comment:
On Fedora, I always add also `--with-system-libmp4v2 --with-system-liba52 --with-system-libmad`. For Fedora, Avidemux is packaged by RPM Fusion which allows build dependencies to be installed by a straightforward

`sudo dnf builddep avidemux`

RPM Fusion needs to be enabled anyway to produce a reasonably feature-complete Avidemux build.

Reply:
Can you extend it? Im not familiar with Fedora :-)

Reply:
I'll do it some day :-)